### DISCUSSION TITLE
Is it possible to enable Markdown or rich text editing for the Description?

### DISCUSSION THREAD

**Discussion by microCloudCode** [medusajs/medusa#4739] [Feature Requests]
Is it possible to enable Markdown or rich text editing for the Description?

**Comment by srindom** [comment]
Nothing prevents you from populating the Product Description field with Markdown text. The trick would be to format the description when you need to display it in your storefront application. E.g., you would do something like this:

```javascript
// my-product-page.jsx

// ...
import { useProduct } from "medusa-react"

export default function({ id }) {
  const { product, isLoading } = useProduct(id)
 

  if (isLoading) {
    return (
      <Skeleton />
    )
  }

  return (
    <div>
        {/* ... */}
        <SomeMarkdownRenderer content={product.description} />
    </div>
  )
}
```

Unfortunately, the description input isn't rendered nicely in the Admin application so understand if the product management doesn't play as nicely as you would want, but you could solve this by adding [an Admin Extension](https://medusajs.com/blog/announcing-admin-extensions/) to the Product Details page that renders product description input.

**Reply by microCloudCode** [reply]
Hello, I've reviewed the documentation regarding Admin Extensions, but I'm still unsure about how to proceed. Adding widgets doesn't seem to address my issue; it somewhat feels disconnected from the current way of editing product descriptions. If I aim to keep the process of editing product descriptions as straightforward as possible, it appears that modifying the source code of the admin application might be the only viable option.

**Reply by StephixOne** [reply]
I think what Seb was suggesting here was to write a widget for the Product Details page that contains a richer/more nicely-rendered way to edit the description, to better-support Markdown.

**Reply by moussaab-moulim** [reply]
the problem is not with rendering the markdown in the storfront 
its haveing a richtext editor in the admin page for product creation
like shopify perexample

**Reply by mahevstark** [reply]
@moussaab-moulim  did you find anything on this?
